src/emulator/functionsEmulator.ts

```diff
@@ -69,19 +69,6 @@ export class FunctionsEmulator implements EmulatorInstance {
     const hub = express();
 
     hub.use((req, res, next) => {
-      // Allow CORS to facilitate easier testing.
```

Comment:
This doesn't seem like a fix for #1275 unless I'm missing something and we handle CORS for Callable elsewhere. I believe the correct behavior should be to drop these headers for `onRequest` but still enable them for `onCall`, am I mistaken?

Reply:
Oh whoops, I see the functions SDK handles it, jk